src/site/_includes/partials/course-index.njk

```diff
@@ -1,9 +1,14 @@
 {%- set courseData = courses[projectKey] -%}
 {%- set pageNavigation = collections.all | navigation(courseData.toc) -%}
-{%- for item in pageNavigation -%}
+{%- for item in pageNavigation.list -%}
```

Comment:
I think the card on the /learn/ page uses the number of items in pageNavigation to indicate how many resources are in the course (note how in the image below it just says ___ resources). So we may need to update that card to count the number of items in pageNavigation.list, or use some other approach to get the total number of  items.

![image](https://user-images.githubusercontent.com/1066253/131928004-7ee23c93-d8b4-4646-ab17-1d9ed47b3896.png)

Reply:
ah I think this is also why percy is failing so when that's fixed that test should pass and unlock the PR

Reply:
Fixed, I missed this one.